### DISCUSSION TITLE
Add switch onChange event for vue

### DISCUSSION THREAD

**Discussion by diego-lipinski-de-castro** [tailwindlabs/headlessui#1212] [Ideas]
React has this, I think in some cases it would be useful for vue too

**Comment by RobinMalfait** [comment]
In Vue you have this as well. In Vue we work with `v-model="foo"`, which is a shorthand for:
```
:modelValue="foo"
@update:modelValue="newValue => foo = newValue"
```

This means that you can already use `@update:modelValue="..."` to get notified when something changes.

**Reply by rahulv3a** [reply]
Thank you so much for pointing it out.

It would be helpful to others if this could be added to the documentation.

**Reply by alvinotienokelly** [reply]
Thank you. It works
